include/aa_pipeline_api.hpp

```diff
@@ -338,6 +338,22 @@ namespace astroaccelerate {
 			}
 		}
 
+		size_t range(){
```

Comment:
This should be get_nRanges

Reply:
Renamed as suggested. Changes in all permitted pipelines